7_react/react1/week3/todo-list-app/src/TodoHeader.js

```diff
@@ -0,0 +1,7 @@
+import React from 'react';
+
+const TodoHeader = () => {
```

Comment:
This is rather subjective and a matter of personal preference, but usually if a component is outputting just a text, it's not necessary to make it a separate component. I.e., you could just use the h2 tag whenever you need to use it. 

Of course, if it were to be more complex and have complex styling and so on, it could be defined as a separate component.